flake.nix

```diff
@@ -136,10 +106,10 @@
         devShells.default = pkgs.mkShell {
           name = "${packageJson.name}-shell";
 
+          inputsFrom = [ self.packages.${system}.default ];
+
           buildInputs = with pkgs; [
             fish
-            nodeEnv.nodeDependencies
```

Comment:
This breaks the development environment as we don't have any dependencies listed in `package.json` available anymore (e.g., `jest`, `prettier`, ...).

Reply:
Ah had an error in thinking, the package does not have any `buildInputs` anymore 😅 
Regardless of the above issue, the required tools would only be `jest` and `prettier`?
`jest` is not available in `nixpkg` 😢

Reply:
Just sticking to `npm install & npm run $CMD` should be acceptable.

Reply:
So nothing to do here, because this should work in the `devShell`?
Or should I add dedicated commands like 
```nix
{
  jest-coverage = self.packages.${system}.default.overrideAttrs (oldAttrs: {
    name = "jest-coverage";
    doCheck = false;
    npmBuildScript = "coverage";
  });
  devShells.default = pkgs.mkShell {
    name = "${packageJson.name}-shell";
    buildInputs = with pkgs; [
      jest-coverage
      ...
    ];
  };
}
```
So that running test is as easy as possible in the `devShells`?

Reply:
That doesn't seem worth the effort 😅 just keep the current change.